ResponsiveWebDesign/styling.css

```diff
@@ -0,0 +1,166 @@
+body {
+	padding-left: 200px;
+}
+
+.head {
+	color: teal;
+	text-decoration: underline;
+	height: 40px;
+	width: 1000px;
+}	
+
+#box {
+	border: 1px;
+	border-style: solid;
+	border-color: orange;
+	width: 960px;
+	padding-left: 0px;
+}
+
+img {
+	width: 200px;
+	height: 200px;
+}
+
+li:nth-child(3n-2){
+	display: inline-block;
+	list-style-type: none;	
+	width: 200px;
+	margin-bottom: 50px;
+	float: left;
+}
+
+li:nth-child(3n-1){
+	display: inline-block;
+	list-style-type: none;	
+	width: 200px;
+	margin: 0 180px 50px 180px;
+}
+li:nth-child(3n){
+	display: inline-block;
+	list-style-type: none;	
+	width: 200px;
+	margin-bottom: 50px;
+	float: right;
+	
+}
+
+@media only screen and (max-width: 1200px) {
+	
+	body {
+		padding-left: 100px;
+	}
+	.head {
+		color: teal;
+		text-decoration: underline;
+		height: 40px;
+		width: 1000px;
+	}
+	#box {
+		border: 1px;
+		border-style: solid;
+		border-color: orange;
+		width: 960px;
+		padding-left: 0px;
+	}
+	img {
+		width: 200px;
+		height: 200px;
+	}
+	li:nth-child(3n-2) {
+		display: inline-block;
+		list-style-type: none;
+		width: 200px;
+		margin: 0 0 50px 30px;
+		float: left;
+		padding-left: 0px;
+	}
+	li:nth-child(3n-1){
+		display: inline-block;
+		list-style-type: none;
+		width: 200px;
+		margin: 0 150px 50px 150px;
+	}
+	li:nth-child(3n){
+		display: inline-block;
+		list-style-type: none;
+		width: 200px;
+		margin: 0 30px 50px 0;
+		float: right;
+	}
+}
+
+@media only screen and (max-width: 992px) {
+
+	body{
+		padding-left: 50px;
+	}
+
+	.head {
+		color: teal;
+		text-decoration: underline;
+		height: 40px;
+		width: 1000px;
+	}
+	#box {
+		border: 1px;
+		border-style: solid;
+		width: 700px;
+		height: 1800px;
+		border-color: orange;
+		padding-left: 0px;
+	}
+	img {
+		width: 200px;
+		height: 200px;
+	}
+	li:nth-child(2n-1){
+		display: inline-block;
+		list-style-type: none;
+		width: 200px;
+		margin: 0 20px 50px 25px;
+		float: left;
+		padding-left: 0px;
+	}
+
+	li:nth-child(2n){
+		display: inline-block;
+		list-style-type: none;
+		width: 200px;
+		margin: 0 25px 50px 0;
+		float: right;
+	}
+}
+
+@media only screen and (max-width: 768px) {
+
+	body{
+		padding-left: 200px;
+	}
+
+	.head {
+		color: teal;
+		text-decoration: underline;
+		height: 40px;
+		width: 1000px;
+	}
+	#box {
+		border: 1px;
+		width: 220px;
+		height: 3000px;
+		border-style: solid;
+		border-color: orange;
+		padding-left: 0px;
+	}
+	img {
+		width: 200px;
+		height: 200px;
+	}
+	li:nth-child(n){
+		list-style-type: none;
+		width: 200px;
+		margin: 0 10px 50px 10px;
+		float: left;
+	}
+	
+}
```

Comment:
Too big a style sheet for this requirement. 

Look at - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
http://programmer.97things.oreilly.com/wiki/index.php/Don't_Repeat_Yourself